src/globals/type/_index.scss

```diff
@@ -17,24 +17,6 @@
 
 // TODO: `2.x` - To be removed.
```

Comment:
Does it make sense to remove the whole file in `2.x` or no?

Reply:
I believe the rest of this work is covered by #255